test/e2e/storage/persistent_volumes.go

```diff
@@ -313,6 +313,32 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				pvc = nil
 			})
 		})
+
+		ginkgo.Context("delete pv prior", func() {
+			ginkgo.BeforeEach(func() {
+				ginkgo.AfterEach(func() {
+					framework.Logf("AfterEach: Cleaning up test resources.")
+					if errs := e2epv.PVPVCCleanup(c, ns, pv, pvc); len(errs) > 0 {
```

Comment:
I think PVPVCCleanup() is probably deleting PVC first.  You need a function that deletes PV first.